README.md

```diff
@@ -53,12 +53,16 @@ actions
    ├── dispatcher
 ├── remote
    ├── cartResolver
+...
+web-src
 ```
 
 The `common` folder contains all the code that fetches and converts 3rd-party data into the GraphQL format. These classes can be used either by the local dispatcher action or by remote resolvers.
 
 The `documentation` folder contains some code that is used to generate a subset of the Magento schema that covers all the queries required by the CIF integration. This is **not** used by the resolvers, see the [Schema documentation](#schema-documentation) section below.
 
+Within the `web-src` folder we provide a mini web application with loads the (GraphiQL)[https://github.com/graphql/graphiql] GraphQL client, which always you to immediately try out the GraphQL actions of the project.
```

Comment:
Within the `web-src` folder we provide a mini web application _which_ loads the [GraphiQL](https://github.com/graphql/graphiql) _(you got the brackets wrong for this link)_ GraphQL client. _It allows_ you to immediately try out the GraphQL actions of the project.